features/step_definitions/trigger.js

```diff
@@ -14,148 +15,227 @@ Before(
     function hook() {
         this.repoOrg = this.testOrg;
         this.repoName = 'functional-trigger';
-        this.pipelineIdA = null;
-        this.pipelineIdB = null;
         this.success_AJobId = null;
         this.fail_AJobId = null;
         this.success_BJobId = null;
         this.fail_BJobId = null;
         this.parallel_AJobId = null;
         this.parallel_B1JobId = null;
         this.parallel_B2JobId = null;
-        this.builds = null;
+
+        this.buildId = null;
+        this.pipelines = {};
     }
 );
 
 Given(
-    /^an existing pipeline on branch "(.*)" with the workflow jobs:$/,
+    /^an existing pipeline on branch "([^"]*)" with the trigger jobs:$/,
```

Comment:
```suggestion
    /^an existing pipeline on branch "([^"]*)" with the workflow jobs:$/,
```
I feel like this should remain a `workflow`.